tests/integration/test_pass_environment_variable.py

```diff
@@ -81,15 +80,15 @@ def run_test(self, mocked_runner, tmpdir, cli_entry, config_fixture, expected):
 
         params = shlex.split(cli_entry) + ["--pp", "never"]
 
-        with mock.patch("sys.argv", params):
-            with mock.patch.dict(os.environ, {"ANSIBLE_NAVIGATOR_CONFIG": cfg_path}):
-                with mock.patch.dict(
-                    os.environ,
-                    {"ANSIBLE_NAVIGATOR_COLLECTION_DOC_CACHE_PATH": coll_cache_path},
-                ):
-                    with mock.patch.dict(os.environ, expected):
-                        with pytest.raises(Exception, match="called"):
-                            cli.main()
+        mocker.patch("sys.argv", params)
```

Comment:
FWIW this could be moved to a reusable fixture depending on `monkeypatch` and `request` with "indirect" to avoid copy-paste in every test.

Reply:
yeah, there's a real opportunity for these to be cleaned up in a future PR.  Right now I'm just focused on the unittest => pytest conversion.

They are passing, although kinda messy, and do test some important runner functionality, so I'm gonna punt on a complete refactor.